packages/ember-application/tests/system/engine_initializers_test.js

```diff
@@ -20,37 +20,37 @@ QUnit.module('Ember.Engine initializers', {
   }
 });
 
-QUnit.test('initializers require proper \'name\' and \'initialize\' properties', function() {
+QUnit.test('initializers require proper \'name\' and \'initialize\' properties', function(assert) {
   MyEngine = Engine.extend();
 
-  expectAssertion(() => {
+  assert.expectAssertion(() => {
```

Comment:
This is causing CI failures, `assert.expectAssertion` isn't "a thing". This should be changed to `expectAssertion`.